src/platforms/platform.dom.js

```diff
@@ -325,6 +331,13 @@ module.exports = {
 	 */
 	_enabled: typeof window !== 'undefined' && typeof document !== 'undefined',
 
+	get useSecureCSS() {
```

Comment:
Does this work?
I would use `Object.defineProperty` instead, to be consistent with other parts of library.
For example: [core.animation.js](https://github.com/chartjs/Chart.js/blob/master/src/core/core.animation.js)

Reply:
Based on https://caniuse.com/#search=ECMAScript%205, Getter and Setter (introduced by ECMAScript 5), are supported at least by IE 10+, Edge, Firefox, Chrome, Safari, Opera (and their mobile variants).

Which ECMAScript version/browser should it support (so I know this for other changes 😃 )? I can also adjust it just for the sake of doing it in a consistent way.